activities/vidispine/items.go

```diff
@@ -0,0 +1,21 @@
+package vsactivity
+
+import (
+	"context"
+	"go.temporal.io/sdk/activity"
+)
+
+type DeleteItemsParams struct {
+	VXIDs []string
+	//	DeleteFiles bool // TODO: Should probably be implemented in the future
+}
+
+// DeleteItems deletes items from Vidispine, including all files on disk!
```

Comment:
Since the param is part of a struct it is in one sense optional, with a default of "false" for bool.
Due to this I don't want to have a `keepFiles` with a default of `false`. Thus `DeleteFiles`.
But I think implementing it is not a big deal so I'll just do it.